lib/ok.ex

```diff
@@ -161,22 +161,39 @@ defmodule OK do
       iex> OK.with do
       ...>   a <- safe_div(8, 2)
       ...>   b <- safe_div(a, 2)
+      ...>   OK.success a + b
+      ...> end
+      {:ok, 6.0}
+
+      iex> OK.with do
+      ...>   a <- safe_div(8, 2)
+      ...>   b <- safe_div(a, 2)
       ...>   {:ok, a + b}
       ...> end
       {:ok, 6.0}
 
       iex> OK.with do
       ...>   a <- safe_div(8, 2)
+      ...>   b = 2.0
+      ...>   OK.success a + b
```

Comment:
ok that makes a lot of sense. The arity 0 point was right in front of me yet I just hadnt seen it :disappointed: